shared/engine/transport-shared.js

```diff
@@ -25,10 +25,10 @@ function _makeOnceOnly (f: () => void): () => void {
 }
 
 // Wrapped to add logging
-function _makeLogged (f: () => void, type: rpcLogType, logTitle: string, extraInfo?: Object): () => void {
+function _makeLogged (f: () => void, type: rpcLogType, logTitle: string, extraInfo?: ?Object, titleFromArgs?: ?Function): () => void {
```

Comment:
add a transform so we can expose the incoming method names. this allows us to search the chrome console for the methods (else they're nested in the request object)